src/javascript/EditPanelContainer/validation/required.spec.js

```diff
@@ -95,5 +95,26 @@ describe('required validation', () => {
                 field4: undefined
             });
         });
+
+        it('should return object with all field with errors when all fields are multiple and required', () => {
```

Comment:
The test about multiple and boolean field is missing.

Reply:
multiple field, boolean or not boolean type it's the same behaviour.

Reply:
Same behaviour ? Why adding code for same beahviour ?

I still think one test is missing here

Reply:
where did you see diff code for multiple ?

If you think/see missing, feel free to add :D